src/SqlPersistence/Saga/RuntimeSagaInfo.cs

```diff
@@ -62,9 +66,23 @@ class RuntimeSagaInfo
         {
             case SqlVariant.MsSqlServer:
                 TableName = $"[{schema}].[{tablePrefix}{tableSuffix}]";
+                FillParameter = ParameterFiller.Fill;
                 break;
             case SqlVariant.MySql:
                 TableName = $"`{tablePrefix}{tableSuffix}`";
+                FillParameter = ParameterFiller.Fill;
+                break;
+            case SqlVariant.Oracle:
+                if (tableSuffix.Length > 27)
+                {
+                    throw new Exception($"Saga '{tableSuffix}' contains more than 27 characters, which is not supported by SQL persistence using Oracle. Either disable Oracle script generation using the SqlPersistenceSettings assembly attribute, shorten the name of the saga, or specify an alternate table name using the SqlSagaAttribute's 'tablePrefix' parameter.");
```

Comment:
Is there a way to centralize these checks somehow in a definition of a SQL dialect? I am pretty sure I've seen this two times already ;-)